mozci/console/commands/push.py

```diff
@@ -438,7 +438,7 @@ def retrigger_failures(
             groups_with_failures.values(), 0, retrigger_limit
         ):
             # If there is more than one task failing in this group, we should retrigger only one of them
-            failing_tasks[0].retrigger(push, count)
+            failing_tasks[0].confirm(push)
```

Comment:
Should we `confirm` in cases it is supported and fall back on `retrigger`?

Reply:
confirm will call retrigger; although the `count` is missing and needs to be dealt with